docs/guide/statusline.md

```
@@ -109,18 +109,18 @@ See [Cost Source Options](#cost-source-options) section for all available modes.
 The statusline displays a compact, single-line summary:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)